sequencer/src/api/endpoints.rs

```diff
@@ -178,16 +168,6 @@ where
                 .body_auto::<Transaction, Ver>(Ver::instance())
                 .map_err(Error::from_request_error)?;
 
-            // Transactions with namespaces that do not fit in the u32
-            // cannot be included in the block.
-            // TODO: This issue will be addressed in the next release.
-            if tx.namespace() > NamespaceId::from(u32::MAX as u64) {
-                return Err(Error::Custom {
-                    message: "Transaction namespace > u32::MAX".to_string(),
-                    status: StatusCode::BadRequest,
-                });
-            }
```

Comment:
Yikes! I removed this code when I switched namespace ID to 8 bytes in the namespace table, but then forgot to restore it when I reverted it back to 4 bytes for serialization compatibility. Thanks for catching it! I'll restore this code as needed pending resolution of https://github.com/EspressoSystems/espresso-sequencer/pull/1499#discussion_r1631308091

Reply:
This code shall remain removed. It's no longer needed because `serde` now enforces this condition as per https://github.com/EspressoSystems/espresso-sequencer/pull/1499#discussion_r1633684374